packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1035,7 +1035,9 @@ export const Dropdown: ComponentWithAs<'div', DropdownProps> &
             tryRemoveItemFromValue();
             break;
           case keyboardKey.Escape:
-            e.stopPropagation();
+            // If dropdown list is open ESC should close it and not propagate to the parent
+            // otherwise event should propagate
+            open && e.stopPropagation();
```

Comment:
Please could you add a test for that? e2e or unit, up to you ;)
Just so we have a tool to test it, in case the behavior changes ;)

Reply:
added